test/blackbox-tests/test-cases/formatting/feature.t/run.t

```diff
@@ -190,7 +190,8 @@ Sometimes, the suggestion is to just remove the configuration.
   5 | (using fmt 1.2)
       ^^^^^^^^^^^^^^^
   Error: Starting with (lang dune 2.0), formatting is enabled by default.
-  To port it to the new syntax, you can delete this part.
+  To port it to the new syntax, you can replace this part by:
+  (formatting (enabled_for dune ocaml reason))
```

Comment:
Unsure whether we should extend the help message by mentioning the differences between > 3.17 dune and 3.17+ 
i.e. that you can remove the stanza, but that it will also format `dune-project` files.